src/utils/src/backend-supports/projects-get-all.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
This file and its functions are not referenced anywhere. This may indicate that the backend functions are doing things manually that should be abstracted here.

Reply:
Let's leave them as examples and file ticket(s) for improvements to the backend.

Reply:
See #411